src/FakeItEasy/A.cs

```diff
@@ -113,6 +113,7 @@ public static T Dummy<T>()
         /// <param name="typeOfDummy">The type of dummy to return.</param>
         /// <returns>A dummy object of the specified type.</returns>
         /// <exception cref="ArgumentException">Dummies of the specified type can not be created.</exception>
+        [EditorBrowsable(EditorBrowsableState.Advanced)]

```

Comment:
At some point you said you wanted all `EditorBrowsable` attributes to be added in a separate commit; maybe I misunderstood. Anyway, I don't mind squashing this commit if you prefer.